fix: document request data (#511)
getsentry/sentry-docs: src/_includes/platforms/python/request-data.md

> Request data is attached to all events: HTTP method, URL, headers, formdata, JSON payloads. Raw bodies and multipart file uploads are excluded and marked as excluded in the event. Personally identifiable information (such as user ids, usernames, cookies, authorization headers, ip addresses) is excluded unless ``send_default_pii`` is set to ``True``.

Reviewer: ```suggestion
Request data is attached to all events: HTTP method, URL, headers, formdata, JSON payloads. Raw bodies and multipart file uploads are excluded and marked as excluded from events. Personally identifiable information (such as user ids, usernames, cookies, authorization headers, ip addresses) are excluded unless ``send_default_pii`` is set to ``True``.
```

Author: So what I was trying to write here is that the event payload will still contain the filename of the uploaded file + a remark that the file content has been omitted from the event intentionally.

Author: I think it's fine now
